CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
   message(STATUS "${PROJECT_NAME} is top level")
 endif()
 
+# options
+option(CAPO_USE_OPENAL "Build and link to OpenAL (otherwise interface is inactive but buildable) (default ON)" ON)
+option(CAPO_VALID_IF_INACTIVE "Have .valid() return true if lib is inactive (default ON)" ON)
```

Comment:
Is `CAPO_VALID_IF_INACTIVE` really required?

Reply:
Otherwise `if (!instance.valid())` returns false, which may not be the desired behaviour; eg a "headless" build where audio behaves like it's working but doesn't actually produce any output. (Ideally graphics should be like that too, but that's levk's problem and I'm probably not going to go that far.)